docs/getting-started/inferno-cli.md

```diff
@@ -25,10 +24,37 @@ The commands available include:
 | `inferno console` | Starts an interactive console session with Inferno. More information can be found on the [Debugging Page](debugging.html#interactive-console). |
 | `inferno help [COMMAND]` | Describes the available commands, or one specific command if specified |
 | `inferno migrate` | Runs database migrations |
+| `inferno new TEST_KIT_NAME` | Create a new test kit. Run `inferno new --help` for additional options. |
 | `inferno services start` | Starts the background services (nginx, Redis, etc.) for Inferno |
 | `inferno services stop` | Stops the background services for Inferno |
 | `inferno start` | Starts Inferno |
-| `inferno suite SUBCOMMAND ...ARGS` | Performs suite-based operations. The available subcommands are `describe`, `help`, and `input_template`.|
+| `inferno suite SUBCOMMAND [...ARGS]` | Performs suite-based operations. The available subcommands are `describe`, `help`, and `input_template`.|
 | `inferno suite help SUBCOMMAND` | View details on the suite subcommands. |
 | `inferno suites` | Lists available test suites |
 {: .grid.command-table}
+
+## Creating a new Test Kit
+
+The `inferno new` command allows you to create a new instance of an Inferno Test
+Kit. This command provides options to specify the names of authors, and load
+required implementation guides locally or from external websites. This will set
+the names of the test folder and Ruby file in the `lib` folder to match the name
+of the test kit, and set the name of the main module. It will also download
+the `package.tgz` file for the specified Implementation Guide(s).
+
+This method can be more convenient than manually cloning the
+[Inferno Template](https://github.com/inferno-framework/inferno-template) repository, as it automatically sets the names of files and folders to match the Test Kit name.  
+It will automatically install the necessary Gem files used by Inferno Core and run the
+database migrations to bring the database up to date. Run `inferno new --help` for more
+options and examples.
+
+Note that this command follows the naming convention used by Ruby gems. The command
+`inferno new MyTestKit` will create a new folder named `my-test-kit`. Inside the 
+`my-test-kit/lib` folder, it will create the subdirectory `my_test_kit` for your test files, 
+and the main Ruby file named `my_test_kit.rb`. The main module in the `.rb` file will be named `MyTestKit`.
+
+In `my-test-kit/spec`, it will create the subdirectory `my_test_kit` and in the main `patient_group_spec.rb` RSpec file, it sets the module name to `MyTestKit::PatientGroup`.
```

Comment:
I would reword this as:
> The new test kit supports [RSpec](https://rspec.info) testing on the test kit itself. It also includes a working example in `my-test-kit/spec/patient_group_spec.rb`

Honestly using RSpec ontop of Inferno is very complicated so I wouldn't expect new users to try this. Using RSpec is more of a "if you know you know" sort of thing.